src/ppx_deriving_yojson.cppo.ml

```diff
@@ -557,11 +561,24 @@ let desu_str_of_type ~options ~path ({ ptype_loc = loc } as type_decl) =
     let poly_type = Typ.force_poly @@ Typ.poly fv @@ ty in
     let var_s = Ppx_deriving.mangle_type_decl (`Suffix "of_yojson") type_decl in
     let var = pvar var_s in
+    let var_s_exn = var_s ^ "_exn" in
+    let { ptype_params; _ } = type_decl in
+    let var_s_exn_args = List.mapi (fun i _ -> argn i |> evar) ptype_params in
+    let var_s_exn_args = var_s_exn_args @ [evar "x"] in
+    let var_s_exn_fun =
+      let rec loop = function
+      | [] -> [%expr match  [%e app (evar var_s) var_s_exn_args] with Ok x -> x | Error err -> raise (Failure err)]
+      | hd::tl -> lam (pvar hd) (loop tl)
+      in
+      loop ((List.mapi (fun i _ -> argn i) ptype_params) @ ["x"])
+    in
```

Comment:
Can you show me the generated code for the _exn parts?

Reply:
Normal:
```ocaml
    let of_yojson_exn x =
      match of_yojson x with | Ok x -> x | Error err -> raise (Failure err) 
    let _ = of_yojson_exn 
```
Polymorphic:
```ocaml
    let of_yojson_exn arg0 arg1 x =
      match of_yojson arg0 arg1 x with
      | Ok x -> x
      | Error err -> raise (Failure err)
    let _ = of_yojson_exn
```